website/static/css/custom.css

```diff
@@ -774,6 +804,18 @@ ul.toc-headings li a:focus {
 }
 
 @media only screen and (max-width: 1023px) {
+    .tocActive .onPageNav {
+        background: #202020;
+        top: 196px;
+    }
+
+    .docsSliderActive .toc section .navGroups {
+        top: 196px;
+    }
+
+    .docsNavContainer {
+        background: #202020;
+    }
 }
```

Comment:
Could you add a style so that `<a>` elements have an underline AND primary color by default.  This will get closer to A11y compliance requiring two methods visual treatment.  Currently, only hover shows and underline and I believe it's better to do the opposite.  Default with underline, hover, without.

Reply:
Fixed.